src/main/java/it/gov/pagopa/pu/bff/mapper/ExportFileRequestDTOMapper.java

```diff
@@ -0,0 +1,61 @@
+package it.gov.pagopa.pu.bff.mapper;
+
+import it.gov.pagopa.pu.bff.dto.LocalDateIntervalFilter;
+import it.gov.pagopa.pu.bff.dto.generated.PaidExportFileRequestDTO;
+import it.gov.pagopa.pu.bff.dto.generated.ReceiptsArchivingExportFileRequestDTO;
+import it.gov.pagopa.pu.bff.util.DateUtils;
+import it.gov.pagopa.pu.processexecutions.dto.generated.OffsetDateTimeIntervalFilter;
+import it.gov.pagopa.pu.processexecutions.dto.generated.PaidExportFileFilter;
+import it.gov.pagopa.pu.processexecutions.dto.generated.ReceiptsArchivingExportFileFilter;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ExportFileRequestDTOMapper {
```

Comment:
I would not create a single mapper... ClassificationExport will have many fields, this class could become very large...
let's create instead a common package where to implement all exportFile related mappers